javascript-bank/bank.js

```diff
@@ -0,0 +1,40 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance <= 0 || Number.isInteger(balance) === false) {
+    return null;
+  }
+  var newAccount = new Account(this.nextAccountNumber, holder);
+  newAccount.deposit(balance);
+  this.accounts.push(newAccount);
+  this.nextAccountNumber++;
+  return newAccount.number;
+};
+
+Bank.prototype.getAccount = function (number) {
+  if (number > this.accounts.length) {
+    return null;
+  }
```

Comment:
This would actually potentially cause problems. An account number is an `id` and those aren't ever meant to be used for math. If accounts ever get deleted, you can end up with an account number larger than the total amount of accounts, so your new customers may not be able to access their accounts. You can just remove it, the `return null` expression below your loop will account for this condition on its own 🤓